crates/red_knot_server/src/server.rs

```diff
@@ -227,6 +227,9 @@ impl Server {
             inlay_hint_provider: Some(lsp_types::OneOf::Right(
                 InlayHintServerCapabilities::Options(InlayHintOptions::default()),
             )),
+            completion_provider: Some(lsp_types::CompletionOptions {
+                ..Default::default()
+            }),
```

Comment:
Nit: We probably want to tinker with the options here in the future

Reply:
Yup I looked, but they all seemed pretty in the weeds. I also noticed that even if I didn't set this, the client still tried to set completion requests. But this seemed the "most correct" given what exists now.